utkdigitalinitiatives/utk_isl_xml_forms
# Phoenix

**JIRA Ticket**: [CON-22](https://jira.lib.utk.edu/browse/CON-22)

# What does this Pull Request do?

This pull request adds an Islandora form for the Phoenix digital collection - https://digital.lib.utk.edu/collections/islandora/object/collections%3Aphoenix

# How should this be tested?

- Create a record using the form
- Update a record*
- Check for any inconsistencies in the resulting XML, including empty nodes

*Updating a record causes an issue with the subject tab panel. When a record is updated, it automatically adds the same number of subjects panes as was added in the initial record. I've added a blank field ("-") in the selection pane so the added subjects are easy to identify. I'm uncertain if this can be fixed.

# Additional Notes:
Currently I have a placeholder in the ISSN field ("XXXX-XXXX"). An ISSN has been applied for for this digitized version of the magazine, but it will not be available for at least a month.

# Interested parties
@markpbaggett 
@CanOfBees
